code/game/objects/structures/xeno.dm

```diff
@@ -616,15 +616,24 @@ TUNNEL
 	max_integrity = 140
 	var/mob/living/carbon/xenomorph/hivelord/creator = null
 
+	hud_possible = list(XENO_TUNNEL_HUD)
+
+
 /obj/structure/tunnel/Initialize(mapload)
 	. = ..()
 	GLOB.xeno_tunnels += src
-
+	prepare_huds()
 
 /obj/structure/tunnel/Destroy()
+	if(!QDELETED(creator))
+		to_chat(creator, "<span class='xenoannounce'>You sense your [sanitize(name)] at [tunnel_desc] has been destroyed!</span>") //Alert creator
+
+	xeno_message("<span class='xenoannounce'>Hive tunnel [sanitize(name)] at [tunnel_desc] has been destroyed!</span>", 2, creator.hivenumber) //Also alert hive because tunnels matter.
```

Comment:
Wasn't there something about the sanitize not being needed for these because you're already forcing the `stripped_input` when assigning the name?